features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/AuthManager.js

```diff
@@ -19,6 +19,7 @@
 import qs from 'qs';
 import CONSTS from 'AppData/Constants';
 import Configurations from 'Config';
+import API from 'AppData/api';
```

Comment:
We have a small problem here, Importing `AppData/api` to Authmanager will import all of `AppData/api`'s dependencies(swagger-js, async-mutex ect) here. Which will increase the initial page load timing.

Ideally, we should have decare the constant outside the API class as a separate entity and export it separately in the `AppData/api` and when importing also , We can use named import to get only the constant values.

Reply:
`AuthManager` is in the initial page load dependency tree, that's why the concern
We have the following options here
- Remove the import and use the string value 
- Change the way constant has defined and export

Reply:
Fixed with fce66217cc5536f61353527c8bfa4e52ccf40ac8